rebound/simulationarchive.py

```diff
@@ -278,7 +278,7 @@ def getBezierPaths(self,origin=None):
         Examples
         --------
         The following example reads in a SimulationArchive and plots
-        the trajectories as Cubic Bezier Curves. It also plots the 
+        the trajectories as Cubic Bézier Curves. It also plots the 
```

Comment:
Because the encoding is not explicitly specified, this fails the tests on travis. I'd suggest leaving it as Bezier.